build.sbt

```diff
@@ -182,7 +183,7 @@ lazy val `acquisition-event-producer` = (project in file("acquisition-event-prod
     libraryDependencies ++= Seq(
       "com.gu" %% "ophan-event-model" % "0.0.17" excludeAll ExclusionRule(organization = "com.typesafe.play"),
       "com.gu" %% "fezziwig" % "1.3",
-      "com.typesafe.play" %% "play-json" % "2.6.14",
+      "com.typesafe.play" %% "play-json" % "2.7.4",
```

Comment:
good, but is there a way to get the play version from the play plugin?  e.g. com.play.SbtPlugin.playVersion or something?

Reply:
that would be nice! It might be possible by accessing `PlayScala.projectSettings`? But that would be quite brittle!

Reply:
Also, the play-json version doesn't actually align with the play plugin version, I've just gone with what our play version depends on